CMR/python/demos/notebooks/vsc.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CMR Python Library\n",
+    "The CMR Python library is a set of tools designed to make using CMR in python easier. Using the\n",
+    "library will relive the calling code from the need to manage HTTP headers, HTTP calls, or deal with\n",
+    "page results. With the library, a calling function simply feeds a map of parameters and gets back\n",
+    "a list of records.\n",
+    "\n",
+    "# Using the Juypter Notebooks with Visual Studio Code\n",
+    "\n",
+    "Visual Studio Code has built in support for Juypter Notebooks. This demonstration will show how to get\n",
+    "started with the CMR Python library to start making queries against CMR, all under Visual Studio Code\n",
+    "\n",
+    "1. Download Visual Studio from https://visualstudio.microsoft.com\n",
+    "1. Launch Visual Studio Code and open a new window.\n",
+    "1. Either pick \"Get Started with Python development\" from the \"Get Started\" screen, or select\n",
+    "`Select Interpreter to Start Juypeter Server`. This action will start the process of loading the\n",
+    "needed software to host a notebook. Once this is done, call the following in the bottom Terminal pane:\n",
+    "\n",
+    "`pip3 install https://github.com/nasa/eo-metadata-tools/releases/download/latest-master/eo_metadata_tools_cmr-0.0.1-py3-none-any.whl`\n",
+    "\n",
+    "This is all that is needed to start using the CMR Python library."
```

Comment:
These directions didn't work for me. I had already previously installed VS Code and installed the necessary extensions for using Jupyter notebooks in VS Code, and indeed done so successfully. So I skipped to step of running the pip3 install command in the bottom terminal -- this worked and said "Successfully installed eo-metadata-tools-cmr-0.0.1" . But then after that when I moved to the first code cell that has just 'from cmr.search import collection', I got "ModuleNotFoundError: No module named 'cmr'".